packages/wp-story-editor/src/components/documentPane/index.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  PublishPanel,
+  ExcerptPanel,
+  SlugPanel,
+  StatusPanel,
+  PageAdvancementPanel,
+  BackgroundAudioPanel,
+  TaxonomiesPanel,
+} from '@web-stories-wp/story-editor';
+
+/**
+ * Internal dependencies
+ */
+import * as publisherLogoCallbacks from '../../api/publisherLogos';
+
+function DocumentPane() {
+  return (
+    <>
+      <StatusPanel />
+      <PublishPanel apiCallbacks={publisherLogoCallbacks} />
```

Comment:
Would it be easier if we'd move this component to `wp-story-editor` as well? Then we don't have to pass callbacks like this.

Reply:
yeah, on it.